multigeiger/multigeiger.ino

```diff
@@ -552,7 +519,6 @@ void loop()
         lastMinuteLog       = current_ms;
       }
     }
-    GMC_counts = 0;  // initialize ISR values
```

Comment:
removed this. it is not needed as the correct value is assigned at the beginning of the loop function and it never needs to be changed until the next loop() invocation.

also, the comment was wrong, this is not an isr value, but a copy of it.